src/Azure.Functions.Cli/Interfaces/IDurableManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+
+namespace Azure.Functions.Cli.Interfaces
+{
+    internal interface IDurableManager
+    {
+        Task DeleteHistory();
+
+        Task GetHistory(string instanceId);
+
+        Task GetRuntimeStatus(string instanceId, bool getAllExecutions);
```

Comment:
Let's remove the `getAllExecutions` parameter since that is not relevant to Durable Functions.